shared/src/main/java/com/faforever/neroxis/mask/FloatMask.java

```diff
@@ -248,7 +302,11 @@ public FloatMask addWhiteNoise(float scale) {
      */
     public FloatMask addWhiteNoise(float minValue, float maxValue) {
         float range = maxValue - minValue;
-        return addPrimitiveWithSymmetry(SymmetryType.SPAWN, (x, y) -> random.nextFloat() * range + minValue);
+        if (symmetrySettings.getSymmetry(SymmetryType.SPAWN).isOddSymmetry()) {
+            return apply((x, y) -> addPrimitiveAt(x, y, random.nextFloat() * range + minValue));
+        } else {
+            return addPrimitiveWithSymmetry(SymmetryType.SPAWN, (x, y) -> random.nextFloat() * range + minValue);
+        }
```

Comment:
This ends up breaking the symmetry guarantees because a new random is used for every x,y